packages/emotion/src/EmotionThemeProvider/index.tsx

```diff
@@ -79,16 +104,15 @@ function EmotionThemeProvider({ children, theme = {} }: any) {
  * If an override object is given, it returns the ancestor theme and
  * the overrides merged together.
  *
- * @param {object} theme - A full theme or an override object
+ * @param {object} themeOrOverride - A full theme or an override object
  * @returns {function} A function that returns with the theme object for the [ThemeProvider](https://emotion.sh/docs/theming#themeprovider-reactcomponenttype)
+ * @module getTheme
  */
-const getTheme = (theme: any) => (ancestorTheme = {}) => {
-  // themeable themes have a 'key' property (= name of the theme),
-  // so without it it's just an overrides objects
```

Comment:
I'd leave this comment above the `isBaseTheme` method so its clear why it works.

Reply:
Done